src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {};
+
+  calculator.result = 0;
+
+  calculator.add = (num) => {
+    calculator.result += num;
+  };
+
+  calculator.subtract = (num) => {
+    calculator.result -= num;
+  };
+
+  calculator.multiply = (num) => {
+    calculator.result *= num;
+  };
+
+  calculator.divide = (num) => {
+    calculator.result /= num;
+  };
```

Comment:
The `divide` method should handle division by zero. Consider adding a check to prevent division by zero, which could result in an error or an unexpected result.